sdks/python/test-suites/portable/py2/build.gradle

```diff
@@ -128,6 +128,7 @@ task crossLanguagePortableWordCount {
         "--shutdown_sources_on_final_watermark",
         "--environment_cache_millis=10000",
         "--expansion_service_jar=${testServiceExpansionJar}",
+        "--sdk_worker_parallelism=1"
```

Comment:
+1 for fixing Python SDK's default value instead of here. BTW does this mean that Flink's cross-language implementation is somehow broken when there is more than one worker process per worker node ?

cc: @mxm

Reply:
@chamikaramj  `WriteToText` relies on shared filesystem, so more than one worker can cause it to fail with the local FS.

If it takes longer to sort out the pipeline option default, then it's probably also OK to merge this first and unblock the CI.

Reply:
Let's add a comment here why this is necessary. Otherwise LGTM.

Reply:
@mxm Done.
@tweise #9829